float64 X_Axis
float64 Y_Axis
float64 Z_Axis
float64 alpha
float64 beta
float64 gamma